pxc-57/Dockerfile.k8s

```
@@ -109,6 +109,20 @@ RUN mkdir -p /etc/mysql/conf.d/ /var/log/mysql /var/lib/mysql /docker-entrypoint
 	&& chown -R 1001:1001 /etc/mysql/ /var/log/mysql /var/lib/mysql /docker-entrypoint-initdb.d \
 	&& chmod -R g=u /etc/mysql/ /var/log/mysql /var/lib/mysql /docker-entrypoint-initdb.d
 
+ARG DEBUG
+RUN [ -n "$DEBUG" ] \
+        && sed -i '/\[mysqld\]/a wsrep_log_conflicts\nlog_error_verbosity=3\nwsrep_debug=1' /etc/mysql/node.cnf \
+        && sed -i 's/exec "$@"/"$@" || sleep infinity/' /entrypoint.sh \
+        && microdnf install -y \
+                gdb \
+                net-tools \
+                nc \
+        && curl -L -o /tmp/telnet.rpm http://vault.centos.org/7.7.1908/os/Source/SPackages/telnet-0.17-64.el7.src.rpm \
```

src rpm has sources, not binaries
```suggestion
        && curl -L -o /tmp/telnet.rpm http://mirror.centos.org/centos/7/os/x86_64/Packages/telnet-0.17-64.el7.x86_64.rpm \
```